includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -555,6 +579,35 @@ public function allowed_items_in_cart() {
 		return true;
 	}
 
+	/**
+	 * Checks whether cart contains a subscription product or this is a subscription product page.
+	 *
+	 * @since   x.x.x
+	 * @version x.x.x
+	 * @return boolean
+	 */
+	public function has_subscription_product() {
+		if ( ! class_exists( 'WC_Subscriptions_Product' ) ) {
+			return false;
+		}
+
+		if ( $this->is_product() ) {
+			$product = $this->get_product();
+			if ( WC_Subscriptions_Product::is_subscription( $product ) ) {
+				return true;
+			}
+		} elseif ( $this->is_checkout() || $this->is_cart() ) {
```

Comment:
I think these are not class methods

```suggestion
		} elseif ( is_checkout() || is_cart() ) {
```

Reply:
🤦‍♂️ Great catch! Fixed in 3304ed8d11407ff8bb2c85c7fd04514fffd67276

Reply:
>  Fixed in 3304ed8

even better 👍